README.md

```diff
@@ -36,6 +36,7 @@ Every crafts person has his/her preferred tools. While you are free to choose wh
 The [Github package manager](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry) preferred as NPM registry. 
 
 - [NVM](https://github.com/nvm-sh/nvm) - Node Version Manager
+- [Working with the NPM registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry)
```

Comment:
Do you think we should combine it with line 36? As a reader, I would think it is a mistake if it is twice the same link.

Reply:
Good point, I will close this PR.